Vic2ToHoI4/Source/HOI4World/MilitaryMappings/AllMilitaryMappings.cpp

```diff
@@ -1,4 +1,5 @@
 #include "AllMilitaryMappings.h"
+#include "../../../../Fronter/commonItems/StringUtils.h"
```

Comment:
No.

Reply:
Dratted autoinclude for headers keeps getting it wrong.